data/types/base/pump/basal_schedule.go

```diff
@@ -24,7 +24,7 @@ func (b *BasalSchedule) Validate(validator data.Validator) {
 func (b *BasalSchedule) Normalize(normalizer data.Normalizer) {
 }
 
-func ParseBasalSchedule(parser data.ObjectParser) *BasalSchedule {
+func parseScheduleItem(parser data.ObjectParser) *BasalSchedule {
 	var basalSchedule *BasalSchedule
 	if parser.Object() != nil {
```

Comment:
Since this function can return nil, do we validate that it isn't nil during the Validate phase?